### PR TITLE
add travis script

### PR DESCRIPTION
**Pull request by videlec**
Provide a straightforward travis script to test under Ubuntu. The miniconda version is discarded for now.

The tests do pass: https://travis-ci.org/videlec/gmpy/builds/268387544

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,47 +1,29 @@
-language: generic
-
-os:
-  - linux
-  - osx
```
Comment (isuruf) [inline .travis.yml]: What's your plan for OSX ?

**Comment by videlec** [comment]
reuse the conda version that was used before with a travis matrix

File: .travis.yml (hunk 1 above)
Comment (videlec) [reply .travis.yml]: reuse the conda version that was used before with a travis matrix...

**Comment by casevh** [comment]
Thanks.

File: .travis.yml (hunk 1 above)
Comment (isuruf) [reply .travis.yml]: @videlec, any progress on OSX?

Comment (videlec) [reply .travis.yml]: @vinklein was supposed to work on it last week.

Comment (vinklein) [reply .travis.yml]: Fixed on PR #167

**Pull request merged**